src/WebJobs.Script/StorageProvider/HostAzureStorageProvider.cs

```diff
@@ -18,17 +17,15 @@ namespace Microsoft.Azure.WebJobs.Script
     /// This serves as a placeholder for a concrete implementation of a storage abstraction.
     /// This StorageProvider provides a wrapper to align all uses of storage by the Functions Host
     /// </summary>
-    internal class AzureStorageProvider : IAzureStorageProvider
+    internal class HostAzureStorageProvider : AzureStorageProvider
```

Comment:
Review/fix the comments above, and detail why we have to override the base here.